### PR TITLE
Media Files

### PR DESCRIPTION
**Pull request by morschm**
This should be how we can store and retrieve files. There might be too much here and less of writing/reading files is needed, or vice-versa.

### CODE REVIEW

**Comment by morschm** [comment]
Just saw the router file was included. Not meant to be there. It will be deleted.

File: file/fileRouter.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+const path = "./allFiles/"
+/*
+Save a file locally by writing it to a local location
+It's path is to a "allFiles" folder, with a subfolder of the clubName
+    and fileName
+Returns the path of the file
+All inputs are strings
+file must be the contents of the file, which can be accessed by calling 
+    retrieveFiles
+*/
```
Comment (zhoutwo) [inline file/fileRouter.js]: You need a function to generate a file path

File: file/fileRouter.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+function postFile(file_id, file) {
```
Comment (zhoutwo) [inline file/fileRouter.js]: Not file_id, but file path. Same for below.

Comment (zhoutwo) [inline file/fileRouter.js]: I assume that should be the request object, or at least some kind of streamable object?

**Review by zhoutwo — CHANGES_REQUESTED** [review]

**Review by zhoutwo — APPROVED** [review]

**Pull request merged**